lib/commands/performance.js

```diff
@@ -45,6 +45,11 @@ async function uploadTrace (localFile, remotePath = null, uploadOptions = {}) {
  * @property {?string} profileName [Activity Monitor] - The name of existing performance profile to apply.
  *                                                      Execute `instruments -s` to show the list of available profiles.
  *                                                      Note, that not all profiles are supported on mobile devices.
+ * @property {?string|number} pid - The ID of the process to meassure the performance for.
```

Comment:
why wouldn't `current` be the default, and maybe have `all` as an option?

Reply:
because this does not work for all profiles, but only for some of them. Also, instruments requires sudo privileges if process id is set and the device is Simulator. This will make our algorithm fail, so I decided it would be safer to keep the current setting